CHANGELOG.md

```diff
@@ -1,7 +1,48 @@
-## Changelog (Current version: 1.3.1)
+## Changelog (Current version: 1.3.2)
 
 -----------------
 
+## 1.3.2 (2016 Apr 20)
+
+### Release Notes
+
+* __FIX__: We discovered a critical issue in the CLI v1.3.1. Version 1.3.1 of the CLI does not return the expected exit code after `bitrise trigger [PATTERN]` if the `trigger` fails. It always returns exit code 0 if the configuration was correct and the workflow was executed, even if a step failed during `trigger`. This version fixes the exit code issue.
```

Comment:
+* __FIX__: although the previous version (1.3.1) fixed the exit code issue for `bitrise run`, the exit code was still not the right one in case of `bitrise trigger`. This version fixes the issue for `bitrise trigger` too, as well as we unified the handling codes of `run` and `trigger` as much as possible.